frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -176,9 +179,20 @@ export const personsLogic = kea<personsLogicType<PersonFilters, PersonLogicProps
                     action = parsedValue !== undefined ? 'updated' : 'removed'
                 }
 
-                actions.setPerson(person) // To update the UI immediately while the request is being processed
-                const response = await api.update(`api/person/${person.id}`, person)
-                actions.setPerson(response)
+                actions.setPerson({ ...person }) // To update the UI immediately while the request is being processed
```

Comment:
note that this was broken before and we were only updating the UI after the request completed